docs/install.md

```diff
@@ -36,6 +37,26 @@ See the [docs on the main page](../README.md) to find the Tekton Dashboard versi
 
 Please, pay attention to annoucements like **deprecated versions** and/or **security related recommendations** when choosing the version you want to install.
 
+## Installing with the installer script
+
+We recently introduced an installer script to make it easy to deploy the Tekton Dashboard with custom options.
+
+While this is still an experimental feature, it should help making the install process customizable and allow you to provide options that suit your needs. We would definitely love to hear your feedback on this new install mode.
+
+You can refer to the dev docs for more infos [on how to use the installer](./dev/installer.md).
+
+**Important note:** this won't work with releases prior to `v0.7.1`, the installer needs specific assets to work and those assets weren't produced in past release thus making the installer impossible to use with versions before `v0.7.1`.
+
+In a nutshell, invoking the `installer` will look something like this:
+
+```bash
+DASHBOARD_VERSION=v0.7.1
```

Comment:
these should be v0.8.0 since we didn't include this for 0.7.1

Reply:
fixed